scripts/influxdb.service

```diff
@@ -12,7 +12,7 @@ LimitNOFILE=65536
 Environment='STDOUT=/dev/null'
 Environment='STDERR=/var/log/influxdb/influxd.log'
 EnvironmentFile=-/etc/default/influxdb
-ExecStart=/bin/sh -c "/usr/bin/influxd -config /etc/influxdb/influxdb.conf ${INFLUXD_OPTS} >> ${STDOUT} 2>> ${STDERR}"
+ExecStart=/bin/sh -c "/usr/bin/influxd -config /etc/influxdb/influxdb.conf ${INFLUXD_OPTS} >> ${STDOUT} 2>>${STDERR}"
```

Comment:
I don't believe there is any problem here. With the space, it works fine for me. In addition, if there is an issue, why not also fix the redirect to `STDOUT`?

Can you show us an example of the issue occuring? I may be missing something.